### DISCUSSION TITLE
install.sh:158 - incredible "shellmanship"

### DISCUSSION THREAD

**Discussion by 0xe10c** [nicknisi/dotfiles#210] [General]
In [install.sh on line 158](https://github.com/nicknisi/dotfiles/blob/bec7787becb45f2f42e4e713e0a1d755a216e8a0/install.sh#L158) you have this line:

`[[ -n "$(command -v brew)" ]] && zsh_path="$(brew --prefix)/bin/zsh" || zsh_path="$(which zsh)"`

I just wanted to say that this is a very elegant use of the bash `&&` and `||` operators.  A lovely little one-liner.

**Comment by nicknisi** [comment]
Thanks!
